packages/terra-icon/scripts/src/generate-icon/renderJsx.js

```diff
@@ -24,7 +25,11 @@ const staticIcons = iconData
 const renderJsx = icon => new Promise((resolve, reject) => {
   let templatePath;
   if (staticIcons.includes(icon.name)) {
-    templatePath = staticIconTemplatePath;
+    if (icon.name.endsWith('LowLight')) {
```

Comment:
We now generate lowlight icons differently than normal static icons because they don't support theming.